src/library/librarycontrol.cpp

```diff
@@ -482,16 +482,26 @@ void LibraryControl::emitKeyEvent(QKeyEvent&& event) {
     VERIFY_OR_DEBUG_ASSERT(m_pLibraryWidget) {
         return;
     }
-    if (!m_pLibraryWidget->hasFocus() && !m_pSidebarWidget->hasFocus()) {
+    if (!QApplication::focusWindow()) {
+        qDebug()<<"Mixxx window is not focused, don't send key events";
+    }
+
+    Qt::Key key = static_cast<Qt::Key>(event.key());
```

Comment:
is this cast necessary to for the compares below? 
If not, I would prefer to keep it as int.
I guess the QT api uses int, in case of values the do not have an enum for. 
The same considerations apply for our code.

Reply:
changed to `bool keyIsTab = event.key() == 16777217;`